libraries/AP_HAL_ChibiOS/hwdef/f303-Universal/hwdef.dat

```diff
@@ -26,3 +26,11 @@ define HAL_PERIPH_ADSB_BAUD_DEFAULT 0
 
 # setup for MSP
 define HAL_MSP_ENABLED 1
+
+define AP_INERTIALSENSOR_ENABLED 1
```

Comment:
i'd prefer a new target for something like MatekL431, MatekL431-OpticalFlow
adding this to f303-universal may consume too much memory/flash and make it no longer work for existing users. The ideal of "universal" firmwares was a mistake from early AP_Periph development